components/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@speechly/react-ui",
   "private": true,
-  "version": "1.0.1",
+  "version": "1.0.2",
```

Comment:
This should be done by the publishing done and may break its workflow, so I suggest reverting this change.